tutorials/commands/flux-job-cancel.rst

```diff
@@ -0,0 +1,122 @@
+.. _flux-job-cancel:
+.. _flux-job-cancelall:
+
+========================
+How to Cancel a Flux Job
+========================
+
+Inevitably submitted jobs will have to be canceled for one reason or another.  This tutorial
+will show you how.
+
+----------------------------
+How to Cancel a Job by Jobid
+----------------------------
+
+The basic way to cancel a job is through ``flux job cancel``.  All you have to do is specify
+the jobid on the command line.  Here is a simple example after submitting a job.
+
+.. code-block:: console
+
+    $ flux mini submit sleep 100
+    ƒh35Dh5qRyq
+
+    $ flux jobs ƒh35Dh5qRyq
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+     ƒh35Dh5qRyq achu     sleep       R      1      1   13.33s corona174
+
+    $ flux job cancel ƒh35Dh5qRyq
+
+    <snip wait a little bit>
+
+    $ flux jobs ƒh35Dh5qRyq
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+     ƒh35Dh5qRyq achu     sleep      CA      1      1   20.18s corona174
+
+In the above example we submitted a simple job via ``flux mini submit`` that simply
+runs ``sleep``.  Passing the resulting jobid to ``flux jobs`` shows that it is
+running (state is ``R``).
+
+We cancel the job simply by passing the jobid to ``flux job cancel``.  After waiting
+a little bit, we see that the job is now canceled in ``flux jobs`` (state is ``CA``).
+
+While we only passed one jobid to ``flux job cancel`` in this example, multiple jobids can be
+passed on the commandline to cancel many jobs.
+
+Note that in this particular example we happened to know the jobid of our job.  If you do
+not know the the jobid of your job, you can always use ``flux jobs`` to see a list of all
+your currently active jobs.
+
+------------------------
+Cancelling All Your Jobs
+------------------------
+
+The ``flux job cancelall`` command allows you to cancel jobs without specifying jobids.
+By default it cancels all of your active jobs, but several options allow you to target a subset of the jobs.
+
+To start off, lets create 100 jobs that will sleep infinitely.  We will use the special ``--cc`` (carbon copy)
+option to ``flux mini submit`` that will submit 100 duplicate copies of the ``sleep`` job.
+
+.. code-block:: console
+
+    $ flux mini submit --cc=1-100 sleep inf
+    <snip - many job ids printed out>
```

Comment:
Optional: could use --quiet to suppress jobids on standard output

Reply:
yeah, I didn't use `--quiet` in the #195 fast job submission guide too.  I just didn't want to have to explain another option I used :P

maybe it's not a net win??

Reply:
Yeah, I only mentioned it since you had to snip the output. Wondering how that will work if output is ever autogenerated. 

I didn't get a chance to review the other tutorial pr